packages/examples/cvat/exchange-oracle/src/handlers/completed_escrows.py

```diff
@@ -282,3 +282,4 @@ def handle_escrows_validations(logger: logging.Logger) -> None:
                 chain_id=chain_id,
                 status=EscrowValidationStatuses.in_progress,
             )
+            session.commit()
```

Comment:
Doesn't session commit automatically?

Reply:
You're right, it does. Leftovers from debugging, removed.